src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -50,21 +52,29 @@ class FooterType extends TranslatorAwareType
      */
     private $router;
 
+    /**
+     * @var ProductPreview
+     */
+    private $productPreview;
```

Comment:
```suggestion
    protected $productPreview;
```

The class is overridable.

Reply:
private $productUrlProvider, 2 lines upper is it not ?